File: 119. Pascal's Triangle II/solution.js
/**
 * @param {number} rowIndex
 * @return {number[]}
 */
var getRow = function(rowIndex) {
  const row = [1];
  for (let i = 1; i < rowIndex + 1; i++) {
    row[i] = (row[i - 1] * (rowIndex - (i - 1))) / i;
  }
  return row;
};

File: 119. Pascal's Triangle II/solution.go
func getRow(rowIndex int) []int {
	if rowIndex == 0 {
		return []int{1}
	}
	lastRow := getRow(rowIndex - 1)
	length := len(lastRow)
	thisRow := make([]int, length+1)
	thisRow[0] = 1
	thisRow[length] = 1
	for i := 1; i < length; i++ {
		thisRow[i] = lastRow[i] + lastRow[i-1]
	}
	return thisRow
}